composer.json

```
@@ -12,13 +12,14 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
+        "php": "8.0.*",

```

Šta misliš ovako? Zadržavamo kompatibilnost sa starijim verzijama (jer realno jeste kompatibilno sve i dalje), a precizniji smo u tvrdnji dokle znamo da smo kompatibilni (do `8.0` garantujemo, možda sve bude okej i za `8.1` pa samo podignemo ovaj constraint kasnije).

Inače `8.0` = `8.0.*` u ovom slučaju, pa možemo da skratimo.

```suggestion
        "php": ">=5.3,<=8.0",

```

---

Ima smisla skroz.